spec/boxr/tasks_spec.rb

```diff
@@ -47,9 +47,10 @@
               expect(task_assignment.resolution_state).to eq('completed')
             }.to raise_error
 
-    puts "delete task assignment"
-    result = BOX_CLIENT.delete_task_assignment(TASK_ASSIGNMENT)
-    expect(result).to eq({})
+    #TODO: {"type":"https://api.box.com/problems","title":"Cannot delete the only assignee of a task","status":400}
+    # puts "delete task assignment"
+    # result = BOX_CLIENT.delete_task_assignment(TASK_ASSIGNMENT)
+    # expect(result).to eq({})
 
     puts "delete task"
```

Comment:
Same w/ this delete. These shouldn't be here?

Reply:
These are all inherited from the original gem.

It's really nice output for the test suite too. It's really slow since it actually makes the API calls.
<img width="502" alt="Screen Shot 2019-08-12 at 5 36 42 PM" src="https://user-images.githubusercontent.com/7145362/62900335-e1baca00-bd27-11e9-92d1-b6dfbcd02ddc.png">

But yeah, we could drop em